src/file.cr

```diff
@@ -2,22 +2,31 @@ require "crystal/system/file"
 
 class File < IO::FileDescriptor
   # The file/directory separator character. `'/'` in Unix, `'\\'` in Windows.
```

Comment:
Shouldn't this comment be changed since `/` is a legal file separator in Unix and Windows?

Reply:
Yeah, these should be changed to just say they're always `/`. This will be changed anyway once we have `Path`.

I'll wait until more comments arrive to fix this.